tbx/impact_reports/blocks.py

```diff
@@ -10,7 +10,7 @@
 
 
 class ImpactReportHeadingBlock(StructBlock):
-    image = ImageChooserBlock(required=False)
+    image = ImageChooserBlock(required=True)
```

Comment:
**Comment(non-blocking)**: This could also be written as

```python
image =  ImageChooserBlock()
```

Reply:
thanks - sorted.